backend/src/test/java/com/github/gadenko/gamereviewsite/backend/controller/GameReviewControllerTest.java

```diff
@@ -0,0 +1,95 @@
+package com.github.gadenko.gamereviewsite.backend.controller;
+
+import com.github.gadenko.gamereviewsite.backend.model.GameReview;
+import com.github.gadenko.gamereviewsite.backend.repo.GameReviewRepo;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class GameReviewControllerTest {
+
+    @Autowired
+    private WebTestClient testClient;
+
+    @LocalServerPort
+    private int port;
+
+    @Autowired
+    private GameReviewRepo gameReviewRepo;
+
+    @BeforeEach
+    public void cleanUp(){
+        gameReviewRepo.deleteAll();
+    }
+
+    @Test
+    void getAllGameReview_isItValid_returnAll() {
+        //GIVEN
+        GameReview gameReview1 = GameReview.builder().title("TES5 Skyrim").headline("Hält der Titel was er verspricht?").gameDescription("Was für eine Fantasy Welt").build();
+        GameReview gameReview2 = GameReview.builder().title("X4: Foundations").headline("Hält der Titel was er verspricht?").gameDescription("Schöner Weltraum Titel. Sehr viele möglickeiten der entfaltung im Space.").build();
```

Comment:
Pls use a new line for every field, when using builder()
```suggestion
        GameReview gameReview1 = GameReview.builder()
        .title("TES5 Skyrim")
        .headline("Hält der Titel was er verspricht?")
        .gameDescription("Was für eine Fantasy Welt")
        .build();
        GameReview gameReview2 = GameReview.builder()
        .title("X4: Foundations")
        .headline("Hält der Titel was er verspricht?")
        .gameDescription("Schöner Weltraum Titel. Sehr viele möglickeiten der entfaltung im Space.")
        .build();
```